tests/testthat/test-error.R

```diff
@@ -242,6 +242,17 @@ test_that("errors in r_session$call() are merged", {
   }
 })
 
+test_that("early error for missing environment file", {
+  tmp <- tempfile()
+  file.rename(env_file, tmp)
```

Comment:
I think it would be better to run the whole test in a background process, so we don't temporarily mess up the current callr session. I.e. putting the whole thing into a function and callr via `callr::r()`

Reply:
@gaborcsardi I moved the test to a function and called it with `callr::r()`. Please let me know if the test can be further improved.